src/components/Match/matchColumns.jsx

```diff
@@ -363,7 +364,12 @@ export const performanceColumns = [
     tooltip: strings.tooltip_lane,
     field: 'lane_role',
     sortFn: true,
-    displayFn: (row, col, field) => strings[`lane_role_${field}`],
+    displayFn: (row, col, field) => (row.is_roaming ?
+      <div>
+        <span>{strings[`lane_role_${field}`]}</span>
+        <span className={subtextStyle.subText}>Roaming</span>
```

Comment:
You could render just this line on `row.is_roaming` which removes the duplication of the lane_role line?